frontend/src/components/dashboard/UserDataTable.vue

```diff
@@ -54,7 +57,7 @@ const filters = ref({
         </div>
 
         <span class="p-input-icon-right">
-            <i class="pi pi-search" />
+            <Search />
```

Comment:
Same comment as previous,  maybe I am confused, is it better to use Icon component?